common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -182,6 +191,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
 
       if (ac != null) {
         ac.setAuthorized(true);
+        RemoteClientUser.set(username);
```

Comment:
1, I think `RemoteClientUser` should be set in the `PlainSaslServer#evaluateResponse` as opposed to here. 2, when the authorization ID and authentication ID are different, which one should you pick for the remote client username?